src/components/Onboarding/steps.ts

```diff
@@ -0,0 +1,261 @@
+          showSkipButton: true,
+        },
+      },
+      {
+        ...getDetails('search-bar'),
+        step: {
+          ...commonStepOptions,
+          target: '#command-bar',
+        },
+      },
+      {
+        ...getDetails('voice-search'),
+        step: {
+          ...commonStepOptions,
+          target: '#voice-search-trigger',
+          placement: 'bottom',
+        },
+      },
+
+      {
+        ...getDetails('juz'),
+        step: {
+          ...commonStepOptions,
+          target: '#juz-tab',
+        },
+      },
+    ],
+    [OnboardingGroup.SETTINGS]: [
+      {
+        ...getDetails('settings'),
+        step: {
+          ...commonStepOptions,
+          target: '#settings-button',
+          showNextButton: false,
+          showSkipButton: true,
+          spotlightClicks: true,
+        },
+      },
+      {
+        ...getDetails('theme'),
+        step: {
+          ...commonStepOptions,
+          target: '#theme-section',
+          spotlightClicks: true,
+          placement: 'left-start',
+          disableScrolling: true,
+        },
+      },
+      {
+        ...getDetails('font-style'),
+        step: {
+          ...commonStepOptions,
+          target: '#quran-font-section',
+          spotlightClicks: true,
+          placement: 'left-start',
+          disableScrolling: true,
+        },
+      },
+      {
+        ...getDetails('font-size'),
+        step: {
+          ...commonStepOptions,
+          target: '#font-size-section',
+          spotlightClicks: true,
+          placement: 'left-start',
+          disableScrolling: true,
+        },
+      },
+      {
+        ...getDetails('wbw-translation'),
+        step: {
+          ...commonStepOptions,
+          target: '#word-by-word-section',
+          spotlightClicks: true,
+          placement: 'left-start',
+        },
+      },
+      {
+        ...getDetails('wbw-transliteration'),
+        step: {
+          ...commonStepOptions,
+          target: '#word-by-word-section',
+          spotlightClicks: true,
+          placement: 'left-start',
+          disableScrolling: true,
+        },
+      },
+      {
+        ...getDetails('wbw-audio'),
+        step: {
+          ...commonStepOptions,
+          target: '#word-by-word-section',
+          spotlightClicks: true,
+          placement: 'left-start',
+          disableScrolling: true,
+        },
+      },
+      {
+        ...getDetails('inline-wbw'),
+        step: {
+          ...commonStepOptions,
+          target: '#word-by-word-section',
+          spotlightClicks: true,
+          placement: 'left-start',
+          disableScrolling: true,
+        },
+      },
+      {
+        ...getDetails('translations'),
+        step: {
+          ...commonStepOptions,
+          target: '#translation-section',
+          spotlightClicks: true,
+          placement: 'left-start',
+        },
+      },
+    ],
+    [OnboardingGroup.READING_EXPERIENCE]: [
```

Comment:
There is an issue when you turn this step on while you are on the reading view:

![Jan-26-2024 12-49-21 am](https://github.com/quran/quran.com-frontend-next/assets/15169499/2da16995-4bc2-4a07-a0d7-7c3522f01a0f)

Reply:
There is another issue, the first step does not work except when you click somewhere and scroll first:

![Jan-26-2024 3-29-49 pm](https://github.com/quran/quran.com-frontend-next/assets/15169499/532e984b-957e-441b-a636-131565fce79c)